connectors/es/client.py

```diff
@@ -21,6 +22,14 @@ class PreflightCheckError(Exception):
     pass
 
 
+class License(Enum):
+    ENTERPRISE = "enterprise"
+    PLATINUM = "platinum"
+    BASIC = "basic"
+    TRIAL = "trial"
+    UNSET = None
+
```

Comment:
ent search also lists `gold`. I see that's discontinued, but IDK what that means in practical terms, or if a current customer could get 8.9.0 on a Gold license.

Reply:
Added `gold`. I suppose the relation is `basic < gold < platinum < enterprise <= trial`?